packages/backend/package.json

```diff
@@ -46,10 +46,10 @@
     "@adonisjs/fold": "^8.2.0",
     "@apollo/client": "^3.3.20",
```

Comment:
Is there a reason you didn't upgrade this too?

Reply:
Missed it, just upgraded it @cairin thx!